src/components/ImageScroller.js

```diff
@@ -41,7 +41,7 @@ const useStyle = makeStyles((theme) => ({
     height: '100%',
   },
   selectedImageCard: {
-    border: `inset ${theme.spacing(1.5)}px ${theme.palette.primary.main}`,
+    // border: `inset ${theme.spacing(1.5)}px ${theme.palette.primary.main}`,
```

Comment:
We need to replace this with another indication of which card is selected.
I'm open to suggestions.